script.sh

```diff
@@ -48,7 +47,7 @@ mkdir db
 
 linkf=~/tools/LinkFinder/linkfinder.py
 
-for i in $(cat $cwd/$1)
+for i in $(cat $1)
```

Comment:
Using pwd doesn't work if an absolute path to the file is given. This happens as it tries to replace the whole path twice, both for the $cwd and $1 which ends up getting appended like /current/path//path/to/file. Thus Removed it and printed the location of the results file insted once the script execution is finished.